.gitignore

```
@@ -3,3 +3,4 @@
 *.svg
 cover.out
 dist/
+node_modules
```

This is a directory, no? How about a trailing slash, just for consistency with the other directory names in this list.

```suggestion
node_modules/
```